website/app/templates/head.hbs

```diff
@@ -21,45 +21,53 @@ We've made some small customizations to simplify this compared to the default te
 
 {{! END GENERAL }}
 
-{{! BEGIN OPENGRAPH }}
+{{! BEGIN TWITTER }}
 
-{{#if this.model.siteName}}
-  <meta property="og:site_name" content={{this.model.siteName}} />
+<meta name="twitter:card" content="summary_large_image" />
+
+{{#if this.model.twitterUsername}}
+  <meta name="twitter:site" content={{this.model.twitterUsername}} />
+  {{!-- <meta name="twitter:creator" content={{this.model.twitterUsername}} /> --}}
```

Comment:
I commented out a few tags to follow @MelSumner's example exactly and got an image to appear (together with adding a full image url).

Would it be ok to just delete these commented out tags? (Or should I try adding any back in which seem important?)

Reply:
If they're not being used we can delete them

Reply:
Thanks, I will go ahead and just delete the ones I commented out.